docs/reference/filters.md

```diff
@@ -25,6 +25,7 @@ Template placeholder is replaced by the value that is looked up in the following
 * request method (`${request.method}`)
 * request host (`${request.host}`)
 * request url path (`${request.path}`)
+* request url rawquery (`${request.rawquery}`)
```

Comment:
I am wondering if we should name it just `${request.query}`

Reply:
I even thought about it but `${request.query.q}` would return a value only and `${request.query}`would return key(s) and value(s) of the query. For me this seemed to be a little inconsistency or even confusing.
What do you think? I am also happy with `${request.query}` if you would prefer?

Reply:
Ok, lets name it raw query - this would be somewhat consistent with lua properties https://github.com/zalando/skipper/blob/master/docs/reference/scripts.md#other-request-fields

But lets name camelcase `rawQuery` to match existing `sourceFromLast` and `clientIP`

Reply:
Lets also describe that it is `encoded request URL query values, without '?'`

Reply:
Thanks for your suggestions @AlexanderYastrebov.
I changed it to `${request.rawQuery}` and tried to describe it as precisely as possible in the documentation.

Reply:
You need to fix tests as well

Reply:
Sorry @AlexanderYastrebov you're right.
I currently fixed the tests.

Reply:
And I fixed the test case `request_rawquery_is_empty` (expected `true` by mistake, instead of `false`).